server/src/main/java/org/opensearch/indices/NodeIndicesStats.java

```diff
@@ -281,7 +390,24 @@ public List<IndexShardStats> getShardStats(Index index) {
      *
      * @opensearch.internal
      */
-    static final class Fields {
-        static final String INDICES = "indices";
+    @PublicApi(since = "2.0.0")
```

Comment:
Had to add `@PublicApi(since = "2.0.0")` for the introduced enum as per the check conditions. Please do let me know if this may cause issues or if there are any concerns over it.